dali/test/python/test_video_pipeline.py

```diff
@@ -117,3 +121,13 @@ def test_multi_gpu_video_pipeline():
     for p in pipes:
         p.build()
         p.run()
+
+# checks if the VideoReader can handle more than OS max open file limit of opened files at once
+def test_plenty_video_files():
+    # make sure that there is one sequence per video file
+    pipe = VideoPipe(batch_size=BATCH_SIZE, data=PLENTY_VIDEO_FILES, step=1000000, sequence_length=1)
```

Comment:
if you incorporate my suggestion of having a configurable max number of open video files then you could test this easily by setting a small number

Reply:
Not really. This `step` is set to make sure we just read one frame per each video as we just want to traverse through all videos. Having a configurable max number won't help either as the real limit is the OS. All we need to test if the limit set inside DALI is smaller than the OS one it works (and it doesn't matter if it is OS limit - 1 or just 1).